src/api/firebase.js

```diff
@@ -115,10 +116,6 @@ export async function updateItem(listId, item) {
 	});
 }
 
-export async function deleteItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to delete an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+export async function deleteItem(listId, item) {
+	return await deleteDoc(doc(db, listId, item.id));
```

Comment:
do you handle failures?

Reply:
I have refactored the code to include try/catch blocks as well as sending an error message to the user on the client side.